content/20.treatments.md

```diff
@@ -941,7 +941,7 @@ There is also some research into possible potential therapeutics or prophylactic
 For example, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
 Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses [@ISBN:9780815332183].
 In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of toll-like receptors TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
```

Comment:
```suggestion
In mouse models, poly(I:C) and CpG, which are agonists of toll-like receptors TLR3 and TLR9, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
```

Reply:
In the particular reference used, 4 agonists were investigated & only poly(I:C) was investigated a lot and showed the most effect. The 2nd most effect was seen with CpG in the small data provided. The least effect was seen with LPS in small data provided. I think maybe the idea of mentioning LPS was because LPS may have a different net relation/effect on immune system as described for example in another reference (the review on trained immunity) but that's not discussed in the particular reference for this sentence and we don't discuss LPS elsewhere in this paragraph.